### DISCUSSION TITLE
No TIF support on Ubuntu 18.04

### DISCUSSION THREAD

**Discussion by Camilochiang** [wkentaro/labelme#1082] [Ideas / Feature requests]
I tried with your newest version and 4.2.9 as recommend  in #675 but in no case i was able to open TIF images. Any suggestion?
It work normally with JPG.

in iPython
```
from` PyQt5.Qt import PYQT_VERSION_STR
PYQT_VERSION_STR
'5.10.1'

QImageReader.supportedImageFormats()
[PyQt5.QtCore.QByteArray(b'bmp'),
 PyQt5.QtCore.QByteArray(b'cur'),
 PyQt5.QtCore.QByteArray(b'gif'),
 PyQt5.QtCore.QByteArray(b'ico'),
 PyQt5.QtCore.QByteArray(b'jpeg'),
 PyQt5.QtCore.QByteArray(b'jpg'),
 PyQt5.QtCore.QByteArray(b'pbm'),
 PyQt5.QtCore.QByteArray(b'pgm'),
 PyQt5.QtCore.QByteArray(b'png'),
 PyQt5.QtCore.QByteArray(b'ppm'),
 PyQt5.QtCore.QByteArray(b'svg'),
 PyQt5.QtCore.QByteArray(b'svgz'),
 PyQt5.QtCore.QByteArray(b'xbm'),
 PyQt5.QtCore.QByteArray(b'xpm')]
```

**Comment by Camilochiang** [comment]
Ok. So after updating qt5 with 
```
sudo apt install qt5-image-formats-plugins
```
I can import the images but I immediately get an error saying the following:

```
Error opening file
Make sure /home/ubuntu/Documents/Research/20210314_Running/20210707_Vine/Data/20210706/Output/Altum/IMG_0001.tif is a valid image file.
Supported image formats: *.bmp,*.cur,*.gif,*.icns,*.ico,*.jpeg,*.jpg,*.mng,*.pbm,*.pgm,*.png,*.ppm,*.svg,*.svgz,*.tga,*.tif,*.tiff,*.wbmp,*.webp,*.xbm,*.xpm
```

It works with TIF files that I did from ARW (RAW files) with three bands (RGB) using the GDAL library, with a single band TIF coming directly out of a multispectral camera, but it faill with a 6 bands GeoTIF (Still a TIF format) .  I will try redoing the 6 band GeoTIF to check if is the number of bands or the additional geo information what generate the problem

**Comment by wkentaro** [comment]
I'm not sure but maybe you're getting an error from here?
https://github.com/wkentaro/labelme/blob/4754b26e733890f42c3f801ef4434e59c53fe17a/labelme/label_file.py#L48-L53
If so, we need somehow change PIL.Image.open to support TIF format.

**Comment by Camilochiang** [comment]
So yes, after checking the problem is `PIL`. It do not have the capacity to read stack TIF pictures as `gdal` have. May be a too specific case to fix it? if not, Would you like to have example file? is quite heavy so I have to share in drive or something.

**Comment by wkentaro** [comment]
I'm happy to support it, but I don't have time to work on that myself.
